container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -221,6 +221,11 @@ configuration:
       secret-generator:
         roles: [configgin-role, secrets-role]
   variables:
+  - name: AEK_PASSWORD
```

Comment:
Does this not need to be immutable? I believe otherwise the existing values probably will be broken when you regenerate secrets.

Reply:
Changed with commit e61a291961c190099d2e2b6269b0ca9a0bdeaee9.
Rebased.